unlock-app/src/hooks/useAutoLogin.ts

```diff
@@ -33,10 +35,14 @@ export function useAutoLogin() {
       } catch (error: any) {
         console.error('Autologin failed.')
         console.error(error)
+        await signOut()
       }
     }
+    if (!canAutoLogin && storedProvider === 'DELEGATED_PROVIDER') {
```

Comment:
Remove the token if autologin fails for `DELEGATED_PROVIDER` ( we keep it for unlock accounts!)